src/components/home/components/specimenTypes/SpecimenTypeFilters.tsx

```diff
@@ -50,6 +51,9 @@ const SpecimenTypeFilters = () => {
         });
     }, []);
 
+    /* Calculate total amount of Specimens */
+    const totalSpecimenCount = (Object.values(disciplines).reduce((accumulator, currentValue) => accumulator + currentValue));
```

Comment:
I believe there is a total value in the API Response as well right? You could use that, so you don't have to calculate in the frontend

Reply:
"meta": {
        "totalRecords": 2241740
    }

Reply:
Did not see that, implemented the total records returned from the API instead of calculating it in the front-end